__test__/index.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env jest */
+
+import whenAllSettled from '../src/.'
```

Comment:
(no need for the `.` at the end, looks a bit funny; technically there's no need for the `/` either, but I think that is fine to keep)